lib/IconButton/IconButton.js

```diff
@@ -195,5 +210,16 @@ module.exports = kind(
 		if (inEvent.originator === this) {
 			this.bubble('onRequestScrollIntoView');
 		}
+	},
+
+	/**
+	* @private
+	*/
+	backgroundOpacityChanged: function (old) {
```

Comment:
Thinking out loud and wondering if this can be abstracted, as the same logic will be shared by both `Button` and `IconButton`, to start.